addons/l10n_id/models/res_bank.py

```diff
@@ -0,0 +1,70 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import requests
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+QRIS_API_URL = "https://qris.online/restapi/qris/show_qris.php"
+
+class ResBank(models.Model):
+    _inherit = "res.partner.bank"
+
+    l10n_id_qris_api_key = fields.Char("QRIS API Key")
+    l10n_id_qris_mid = fields.Char("QRIS Merchant ID")
```

Comment:
We'll need to reevaluate the other module (emv) on this, but I think this is not very safe and goes against the safety mechanism we put in place last time (see allow_out_payment)

tldr; fields should be readonly when allow_out_payment is true. this should be enforced in the write, similar to what's done in res_partner_bank in the account module